### DISCUSSION TITLE
Convert unknown labels to yolov5

### DISCUSSION THREAD

**Discussion by elanI337** [ultralytics/yolov5#5895] [Q&A]
Hello datascience-community,

i need your kind assistance.

I own a dataset of  images with unknown label format, wich is:

**angry_actor_104.jpg 0 28 113 226 141 22.9362 0**

It indicates an image as follows:

**image_name face_id_in_image face_box_top face_box_left face_box_right face_box_bottom face_box_cofidence expression_label**

My question is: How can this be **converted into the yolov5 format**?

I have been looking this up for a long time and hope someone can help.
Thank you very much in advance.

Best regards
Phil

**Comment by glenn-jocher** [comment]
@elanI337 see Train Custom **Data** tutorial for YOLOv5 dataset guidelines:

<img width="986" alt="Screenshot 2021-12-06 at 17 39 57" src="https://user-images.githubusercontent.com/26833433/144885844-2ea76abc-cff9-43df-af4a-f79b88cc285c.png">


## YOLOv5 Tutorials

* [Train Custom Data](https://github.com/ultralytics/yolov5/wiki/Train-Custom-Data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://github.com/ultralytics/yolov5/wiki/Tips-for-Best-Training-Results)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://github.com/ultralytics/yolov5/issues/2518)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://github.com/ultralytics/yolov5/issues/475)
* [PyTorch Hub](https://github.com/ultralytics/yolov5/issues/36)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://github.com/ultralytics/yolov5/issues/251) 🚀
* [Test-Time Augmentation (TTA)](https://github.com/ultralytics/yolov5/issues/303)
* [Model Ensembling](https://github.com/ultralytics/yolov5/issues/318)
* [Model Pruning/Sparsity](https://github.com/ultralytics/yolov5/issues/304)
* [Hyperparameter Evolution](https://github.com/ultralytics/yolov5/issues/607)
* [Transfer Learning with Frozen Layers](https://github.com/ultralytics/yolov5/issues/1314)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
